app/controllers/api/v1/plants_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::PlantsController < ApiController
+
+  def create
+    plant_data = params["plantData"]
+    user = current_user
+
+    plant = Plant.new(api_id: plant_data["id"], name: plant_data["name"], image: plant_data["image"])
```

Comment:
not required, but you can still use strong_params for this sort of thing if you like! DRYs up the code and adds a smidge of security